polartools/load_data.py

```diff
@@ -208,3 +216,256 @@ def is_Bluesky_specfile(source, folder=""):
+                    row.append(item[0]["args"][-2])
+                    row.append(item[0]["args"][-1])
+
+            elif key == "time":
+                time = []
+                for t in item:
+                    time.append(
+                        datetime.fromtimestamp(t).strftime("%m/%d/%Y %H:%M:%S")
+                    )
+                row.append(time)
+
+            else:
+                if None in item:
+                    item.remove(None)
+                if len(item) == 1:
+                    item = item[0]
+                row.append(item)
+        table.rows.append(row)
+
+    print(table.reST(fmt=table_fmt))
+
+
+def collect_meta(scan_numbers, db, meta_keys, query=None):
+    """
+    Extracts metadata of a list of scans.
+
+    Parameters
+    ----------
+    scans : iterable
+        Scan numbers or uids.
+    db : databroker database
+        Searcheable database
+    scan_to : int, optional
+        Final scan number to process. Note that this is only meaningful if
+        an integer is passed to `scans`.
+    meta_keys : iterable
+        List with metadata keys to read.
+    query : dictionary, optional
+        Search parameters.
+
+    Returns
+    -------
+    meta : dictionary
+        Metadata organized by scan number or uid (whatever is given in
+        `scans`).
+    """
+    db_range = db_query(db, query=query) if query else db
+    output = OrderedDict()
+    for scan in scan_numbers:
+        try:
+            start = db_range[scan].metadata["start"]
+            stop = db_range[scan].metadata.get("stop", None)
+
+            output[scan] = OrderedDict()
+            for key in meta_keys:
+                output[scan][key] = [start.get(key, None)]
+                if stop is None:
+                    output[scan][key] += [None]
+                else:
+                    output[scan][key] += [stop.get(key, None)]
+                output[scan][key] = _flatten_list(output[scan][key])
+
+        except KeyError:
+            warn(f"The scan number {scan} was not found.")
+
+    return output
+
+
+def _flatten_list(inp):
+    """Only handles lists to second level"""
+    output = []
+    for item in inp:
+        if isinstance(item, list):
+            output += [i for i in item]
+        else:
+            output.append(item)
+    return output
+
+
+def lookup_position(db, scan, search_string="", query=None):
+    """
+    Lookup positioner values in past scans.
+
+
+    Parameters
+    ----------
+
+    db : databroker database
+        Searcheable database
+    scan : integer
+        Scan numbers or uids.
+    search_string : string
+        Full or part of positioner name.
+    query: dict
+        Search parameters.
+
+
+    Returns
+    -------
+    output: list
+
+    """
+
+    db_range = db_query(db, query=query) if query else db
+
+    status = db_range[scan].metadata["stop"]
+    if status and status["exit_status"] == "success":
+        baseline = load_databroker(scan, db, "baseline")
```

Comment:
We need to be a bit careful here. It turns out, the way that the `databroker.v1` and `databroker.v2` load the `dataframe` is different (if the v2 is used, this function will not work). So I suggest that you replace line 448 to:
```python
# Enforces use of v1. Will need to update this function if v2 loads faster in the future.
baseline = load_databroker(scan, db, "baseline", use_db_v1=True)
```